styles/style.scss

```diff
@@ -0,0 +1,166 @@
+
+*{
+    box-sizing: border-box ;
+}
+.container{
+  position: relative;
```

Comment:
@sandesh9988 Style.scss khali chodnu parcha.
Create files for everything you want to write ani tesma lekha. Style.scss ma sake samma kei nalekha. 

Example: https://github.com/bishaller/jug